website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -92,10 +141,23 @@ Authentication, then your aws credentials are likely misconfigured. Try running
 `aws configure` to set up AWS access keys, and pick a default region. If you have any questions,
 please refer to the official AWS documentation on [Configuration and credential file settings](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
 
+
+| Profile field | Example | Description |
+| ------------- | ------- | ------------ |
+| `method` |IAM| use IAM to authenticate |
+| `iam_profile` | analyst | dbt will use the specified profile from your ~/.aws/config file |
+| `cluster_id` | CLUSTER_ID| Required for IAM |
+| `user`   | username | Account user to log into your cluster |
```

Comment:
might also be worth adding a sentence above mentioning when exactly `region` is required. it's only needed for auth to serverless clusters, right?
```suggestion
| `user`   | username | Account user to log into your cluster |
| `region`  | us-east-1 | Optional | 
```

Reply:
Region is used for IAM auth as per this [doc](https://github.com/aws/amazon-redshift-python-driver/blob/master/tutorials/001%20-%20Connecting%20to%20Amazon%20Redshift.ipynb)